crates/project_model/src/workspace.rs

```diff
@@ -824,6 +826,7 @@ fn add_target_crate_root(
         file_id,
         edition,
         Some(display_name),
+        Some(pkg.version.clone()),
```

Comment:
I tried implementing this in the exact same manner, but this was the thing I didn't find, great!

Reply:
Oh I'm sorry, I didn't mean to take this contribution opportunity away from you

Reply:
For future reference if you are interested in working on things you also submit issues for feel free to drop a note. In that case we can also immediately drop some mentoring notes if requested.

Reply:
No that's fine, I gave up and filed an issue to let someone else take it :)